### ISSUE TITLE
[BUG]: Spider-Man (2002) (SLUS-20336) web swinging sound plays inconsistently

### ISSUE THREAD

**Issue by TheTechnician27** [PCSX2/pcsx2#10969]
### Describe the Bug

In Spider-Man (2002), each time you launch a web, a sound plays to indicate that a web has been deployed. On PCSX2, this web sound effect frequently fails to play.* This also happens in the main menu, where switching menu items is supposed to play the sound effect. Finally, this also happens on the pause menu, although it is slightly more difficult to pick up on because another sound effect plays concurrent to it. [A playthrough of the game](https://www.youtube.com/watch?v=LfEqPvydZV0) – definitely played on original hardware** – shows no indication of this happening, let alone as often as it does on PCSX2.

This unfortunately seems largely random, but it occurs considerably more often in the main menu than in-game. This occurs both when slow-swinging (pressing R2 and letting go) and when fast-swinging (holding R2), but it seems it may happen slightly less often when fast-swinging (this one is highly anecdotal).

Note 1: There is a 'swooshing' sound effect that plays when launching the web, but this is not played in place of the web swing sound effect. It is intended to be played subtly alongside the web swing sound, but because the latter is not present, it is the only sound that plays. It can be seen that this is not a corruption of the sound effect by using the main menu, where no sound effect plays whatsoever where one is intended.

Note 2: The linked gameplay is definitely using the PlayStation version, as when the player collects button combos, they correspond to the PlayStation face buttons. This was definitely not played on PCSX2, as it was uploaded in October 2013. Before 2024, there was a glitch whereby the lens flare caused by the Sun (always in a fixed location on the first level) would show through buildings. Multiple camera angles are shown where this would definitely occur on previous versions of PCSX2, and it does not occur. It definitely also was not played on Purei, as even three years ago, [the game had extremely conspicuous issues.](https://www.youtube.com/watch?v=46gBLhKVbxk) This game was never remastered or re-released for future PlayStation consoles, meaning the only other way to play it at the time would have been via backwards compatibility on PlayStation 3.

### Reproduction Steps

The absolute easiest way to do this is in the main menu by using the left and right directional buttons to change selections. You can also go into the pause menu and use the up and down directional buttons to change your selection, although this seems to happen more rarely. However, if you insist on finding it in-game: enter the first level. Start swinging, then line yourself up with one of the major roadways. As he hits the boundaries of the level, Spider-Man will automatically turn around. Thus, he will continue swinging along this path effectively indefinitely because he "bounces" back and forth. Because the frequency at which Spider-Man deploys the webs is effectively fixed, it is extremely easy to notice when one is missing. So maybe just let this play out for a few minutes, at which point you're likely to hear a few instances of this.

### Expected Behavior

To my understanding, the web-swinging sound effect should always play when you deploy your web or change your selection in the main menu.

### PCSX2 Revision

v1.7.5637

### Operating System

Linux (64bit) - Specify distro below

### If Linux - Specify Distro

Manjaro – Flatpak

### CPU

Ryzen 7 5800X

### GPU

GTX 1070

### GS Settings

Default

### Emulation Settings

Default

### Logs & Dumps

Nothing shows up in the console log when this happens.

**Comment by TheTechnician27** [comment]
[SpiderTAS.p2m2.tar.gz](https://github.com/user-attachments/files/18350760/SpiderTAS.p2m2.tar.gz)

This save state (taken on a clang-devel build of v2.3.95) combined with the input recording reliably reproduces the issue.

**Comment by TheTechnician27** [comment]
Willing to call this bug squashed. I've been messing around in the first level for nearly half an hour, and I haven't been able to trigger the game even once. Testing on #13497, but I haven't tested against latest nightly as a control. @PeterDelta if you'd like to give it another try, I think we can close this.

**Comment by PeterDelta** [comment]
Yes, the sound is playing without delay as expected. I'd say it's fixed, thanks :)
